playbooks/roles/thoth-middletier/tasks/main.yaml

```
@@ -30,9 +30,13 @@
         -p THOTH_CEPH_BUCKET="{{ THOTH_CEPH_BUCKET }}"
     ignore_errors: true
 
-  - name: deploy Thoth Results API
+  - name: deploy Thoth Result API
     command: oc new-app --template="{{ THOTH_INFRA_NAMESPACE  }}/thoth-result-api-deployment"
 
+  - name: deploy Thoth Result API BuildConfig
+    command: oc new-app --template="{{ THOTH_INFRA_NAMESPACE  }}/thoth-result-api-buildconfig"
+    ignore_errors: true
```

Why do you want to ignore errors?